openpdf/src/main/java/com/lowagie/text/Cell.java

```diff
@@ -851,4 +863,21 @@ public void setRight(int value) {
 		throw new UnsupportedOperationException(MessageLocalization.getComposedMessage("dimensions.of.a.cell.are.attributed.automagically.see.the.faq"));
 	}
 
+	@Override
```

Comment:
Can you add JavaDoc to this and any other new methods?

Reply:
I have not added Javadoc here because it's an implementation of interface method which is documented. I can add `@inheritDoc`, I guess. What do you think?